### PR TITLE
Tupha/disallow strings for ints nodejs

### PR DESCRIPTION
**Pull request by tuanh118**

### CODE REVIEW

File: source/nodejs/adaptivecards/src/card-elements.ts [hunk 1]
```diff
@@ -990,8 +990,10 @@ export class TextBlock extends CardElement {
         this.color = Utils.getEnumValueOrDefault(Enums.TextColor, json["color"], this.color);
         this.isSubtle = json["isSubtle"];
         this.wrap = json["wrap"] === undefined ? false : json["wrap"];
+        if (typeof (json["maxLines"]) === "number") {
```
Comment (dclaux) [inline source/nodejs/adaptivecards/src/card-elements.ts]: You seem to have an indentation problem, the body of the if block isn't indented properly.

Comment (tuanh118) [reply source/nodejs/adaptivecards/src/card-elements.ts]: yeah my commit editor was acting weird. I also fixed to use the current `typeof` practice

**Comment by dclaux** [comment]
Hopefully that won't break any existing scenario...

**Comment by tuanh118** [comment]
@dclaux I checked all the examples we have in https://adaptivecards.io. All of them use integer `maxLines`. We can look into building a test suite to compare final html using Jest if we don't have anything built in already

**Pull request merged**